classes/class-bsf-custom-fonts.php

```diff
@@ -40,11 +40,10 @@ public function __construct() {
 			require_once BSF_CUSTOM_FONTS_DIR . 'includes/class-bsf-custom-fonts-taxonomy.php';
 			require_once BSF_CUSTOM_FONTS_DIR . 'classes/class-bsf-custom-fonts-render.php';
 
-			if ( is_admin() ) {
-				require_once BSF_CUSTOM_FONTS_DIR . 'includes/class-bsf-custom-fonts-admin.php';
-				require_once BSF_CUSTOM_FONTS_DIR . 'classes/class-bsf-custom-fonts-white-label.php';
-			}
-
+			// if ( is_admin() ) {
```

Comment:
Remove commented code if not required @imnavanath

Reply:
Updated